samples/AspNetIntegration/SimpleHttpTrigger/SimpleHttpTrigger.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+
+namespace AspNetIntegration
+{
+    public class SimpleHttpTrigger
+    {
+        //<docsnippet_aspnet_http_trigger>
+        [Function("SimpleHttpTrigger")]
+        public IActionResult Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequest req)
+        {
+            return new OkObjectResult("Welcome to Azure Functions!");
```

Comment:
Should we also have an example which show how to get HttpContext and use that? Ex: Read a header from httpcontext?

Reply:
This is a smaller PR to create the app and add bits needed for initial docsnippets - more scenarios can be tracked through issues and added to this kind-of epic: https://github.com/Azure/azure-functions-dotnet-worker/issues/1656

Reply:
My bad. Did not realize we were using `HttpRequest` type here. I thought it was `HttpRequestData`